### PR TITLE
add Kyverno mentorships

### PR DESCRIPTION
**Pull request by chipzoller**
Signed-off-by: Chip Zoller <chipzoller@gmail.com>

### CODE REVIEW

File: programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md [hunk 1]
```diff
@@ -48,4 +48,33 @@
...
 
+### Kyverno
+
+#### Pod Security Admission Integrations
+
+- Description: Integrate Kubernetes Pod Security with Kyverno - Part Ⅲ
+- Expected Outcome: Kyverno's podSecurity "subrule" has the ability to exclude based on specific field paths and not just the control level.
+- Recommended Skills: Golang, Kubernetes, Pod Security
+- Mentor(s):
+  - Mentor Name (@realshuting, shuting@nirmata.com)
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:59]:
```suggestion
  - Shuting Zhao (@realshuting, shuting@nirmata.com)
```

File: programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md [hunk 2]
```diff
@@ -48,4 +48,33 @@
...
+- Upstream Issue: https://github.com/kyverno/kyverno/issues/6144
+
+#### Policy Exceptions 2.0
+
+- Description: Enhancements for Kyverno Policy Exceptions including support for images and conditions.
+- Expected Outcome: A future version of Kyverno has enhanced support for its Policy Exception resource.
+- Recommended Skills: Golang, Kubernetes
+- Mentor(s):
+  - Mentor Name (@jimbugwadia, jim@nirmata.com)
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:68]:
```suggestion
  - Jim Bugwadia (@jimbugwadia, jim@nirmata.com)
```

File: programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md [hunk 3]
```diff
@@ -48,4 +48,33 @@
...
+- Upstream Issue: https://github.com/kyverno/kyverno/issues/7907
+
+#### Kyverno Kuttl Enhancements
+
+- Description: Add new features to the kuttl application used by the Kyverno project to aid in its end-to-end testing process.
+- Expected Outcome: Kyverno's fork of kuttl has these enhancements allowing more and better test cases to be written.
+- Recommended Skills: Golang
+- Mentor(s):
+  - Mentor Name (@eddycharly, charles.edouard@nirmata.com)
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:77]:
```suggestion
  - Charles-Edouard Brétéché (@eddycharly, charles.edouard@nirmata.com)
```

**Review by nate-double-u — APPROVED** [review]
Thanks for this @chipzoller.

Couple suggestions inline, it helps me to have the mentor's names in the project when i'm adding everything into the LFX system.

**Comment by chipzoller** [comment]
Argh...sorry about that!

**Review by nate-double-u — APPROVED** [review]
Thanks for the update @chipzoller, and thanks for mentoring @realshuting, @JimBugwadia, @mariamfahmy98, and @eddycharly!

**Pull request merged**